### ISSUE TITLE
Choose item hightlight color

### ISSUE THREAD

**Issue by aljibbs** [Cilestal/clickable_list_wheel_view#10]
Is there a way to change the background color of the selected item or active item in the scroll viewport ?

**Comment by Cilestal** [comment]
Yes you can use index from onSelectedItemChanged method. For the item at the given index, change the background and other options.

**Comment by Cilestal** [comment]
If you need to add a hightlight color on click, you can copy the contents of the clickable_list_wheel_widget.dart file into your code, and replace the GestureDetector on line 78. For example, you can replace it with InkWell.

```
  return InkWell(
       onTap: _onTap,
       onTapUp: (tapUpDetails) {
         _tapUpDetails = tapUpDetails.localPosition;
       },
       child:widget.child,
     );
```

**Comment by aljibbs** [comment]
> Yes you can use index from onSelectedItemChanged method. For the item at the given index, change the background and other options.

I'll be grateful to see an example of this please. I'm still not able to see how to implement this.
